### PR TITLE
Add images for "Prezzi-zonali-GME" custom integration

### PR DESCRIPTION
**Pull request by Gariam-1**
## Type of change

- [ ] Add a new logo or icon for a new core integration
- [ ] Add a missing icon or logo for an existing core integration
- [x] Add a new logo or icon for a custom integration (custom component)
  - [x] I've added a link to my custom integration repository in the PR description
  - [ ] I've opened up a PR for my custom integration on the [Home Assistant
    Python wheels repository](https://github.com/home-assistant/wheels-custom-integrations)
- [ ] Replace an existing icon or logo with a higher quality version
- [ ] Replace an existing icon or logo after a branding change
- [ ] Removing an icon or logo

## Additional information

- This PR fixes or closes issue: fixes #
- Link to code base pull request: https://github.com/Gariam-1/brands
- Link to documentation pull request: 
- Link to integration documentation on our website: 
- Link to custom integration repository: https://github.com/Gariam-1/Prezzi-zonali-GME

## Checklist

- [x] The added/replaced image(s) are **PNG**
- [x] Icon image size is 256x256px (`icon.png`)
- [x] hDPI icon image size is 512x512px for  (`icon@2x.png`)
- [x] Logo image size has min 128px, but max 256px, on the shortest side (`logo.png`)
- [x] hDPI logo image size has min 256px, but max 512px, on the shortest side (`logo@2x.png`)

### CODE REVIEW

**Review by frenck — APPROVED** [review]
Hi there @Gariam-1 👋 

It seems like the upload image isn't a valid image (chance the CI is also failing). Can you take a look?

Thanks! 👍 

../Frenck

**Review by frenck — CHANGES_REQUESTED** [review]
Oh wrong button... early morning and in need of ☕ 😬 

Let me try again...

**Comment by Gariam-1** [comment]
> Hi there @Gariam-1 👋
> 
> It seems like the upload image isn't a valid image (chance the CI is also failing). Can you take a look?
> 
> Thanks! 👍
> 
> ../Frenck

Hi, I initially brached this by creating an empty file to create the pzo_sensor directory, then uploaded the images [here](https://github.com/Gariam-1/brands) thinking they were the same place, but apparently they are not.

Now I've uploaded them correctly, but it's complaining about the x2 files. I don't know what's wrong with them, they are all png 512px. The only thing is that I used an AI upscaler for them, can this be the issue?

Thanks.

**Comment by frenck** [comment]
CleanShot 2024-12-15 at 20 16 45

The correct filename is ending on `@2x` not `@x2`...

../Frenck

**Comment by Gariam-1** [comment]
> The correct filename is ending on `@2x` not `@x2`...

Oh, of course it has to be the classic stupid mistake... sorry about that.

**Review by frenck — APPROVED** [review]
Thanks, @Gariam-1 👍 

../Frenck
